review-me/review-me.js

```diff
@@ -0,0 +1,31 @@
+// hello.js
+
+var message = "Hello, world!"
+console.log(MESSAGE);
+
+function printmessage(msg) {
+    if(msg != null){
```

Comment:
7  :  Ine fficient  null  check  Potential ly  allows  empty  strings  or  undefined ;  better  to  use  msg  ==  null